packages/vit-s/src/TitleInfo.js

```diff
@@ -57,11 +57,13 @@ function PlotOptions(props) {
   const { options } = props;
   const [open, setOpen] = useState(false);
   const classes = useStyles();
+
+  const buttonIcon = useMemo(() => (<SettingsIconWithArrow open={open} />), [open]);
```

Comment:
Would React.memo-ing the SettingsIconWithArrow component itself work as an alternative option?